ezidapp/models/identifier.py

```diff
@@ -1146,6 +1147,15 @@ def computeComputedValues(self):
 
 class Identifier(IdentifierBase):
 
+    @property
+    def search_identifier(self):
```

Comment:
Is the `search_identifier` property used somewhere?

Reply:
I don't believe it is used anymore.  It was used at one point when there was more code in there for handling some complexities of things that we removed.